packages/components/src/count/docs/example.jsx

```diff
@@ -1,15 +0,0 @@
-const count = () => {
```

Comment:
Curious to know how this broke the unit test?

Reply:
Good question -- I saw failing unit tests and misunderstood the issue. I removed the example.jsx because it seemed unnecessary, but reintroduced it in https://github.com/Automattic/wp-calypso/pull/80409/commits/e533a6a5a618b92f5810b8f2cf553518bc5f2575.